Dangerfile

```diff
@@ -6,9 +6,21 @@ module VSPDanger
       prepare_git
 
       [
+        GemfileProtector.new.run,
         ChangeLimiter.new.run,
         MigrationIsolator.new.run
-      ]
+      ].sort(&method(:severity_sort))
+    end
+
+    # expects severities of :error > :warning > :info
```

Comment:
Why do we have to warn in this order?

Reply:
This PR solves it https://github.com/department-of-veterans-affairs/vets-api/pull/5779

So I'll take it out of here.  But we were doing a warn/fail based on the order of the array.  Which would mean they would have gotten a warning for 200+ lines of code and never saw a fail for changing migrations.

Reply:
Also, I misunderstood `fail`.  Apparently Dangerfile will return all warns and fails.